source/lang/number.d

```diff
@@ -60,14 +60,14 @@ struct MpfrBigNumber
 
     pragma(inline, true) this(SmallNumber other)
     {
-        mpfr_init2(mpfr, 80);
+        mpfr_init2(mpfr, 128);
         mpfr_set_d(mpfr, other, mpfr_rnd_t.MPFR_RNDN);
     }
 
     pragma(inline, true) static MpfrBigNumber empty()
     {
         MpfrBigNumber ret = void;
-        mpfr_init2(ret.mpfr, 256);
+        mpfr_init2(ret.mpfr, 128);
```

Comment:
Wouldn't it make sense to make 128 a const global, so it doesn't have to be changed in two places (could also be passed as a commandline parameter? - not now though lol)

Reply:
in dlang there is something known as a manifest constant (really just compile time) one of these could be used or a command flag. Thanks!